src/graphnet/data/dataconverter.py

```diff
@@ -435,6 +437,8 @@ def _extract_data(self, fileset: FileSet) -> List[OrderedDict]:
                     continue
             if self._skip_frame(frame):
                 continue
+            if self._filter_mask(frame, self._I3filters):
```

Comment:
`self._filter_mask` will return `False` by default `if "FilterMask" not in frame`. That means that everyone will be unable to extract i3frames without this field. That's not good!

You can resolve this by either

A) Make `_filter_mask` return `True` by default in these cases
B) change `if self._filter_mask(frame, self._I3filters)` to `if self._filter_mask(frame, self._I3filters) & len(self._I3filters) > 0`.

Reply:
@Aske-Rosted

Reply:
It should very much work such that a `True` return from the _filter_mask call will skip the frame (through a continue call) while a `False` return will move on to include the frame. I agree this is a little counter intuitive and should probably be specified in the description.

Reply:
Sorry. I was fooled. You are right!